src/clr-addons/quick-list/quick-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div [ngClass]="onCard?'card':''">
+  <div class="clr-row">
+    <div *ngIf="!!label" class="clr-col-4">
```

Comment:
implement the label projection like in combobox, that way classes can be controlled by the user